doc/api-examples/python/setup.md

```diff
@@ -0,0 +1,16 @@
+<!-- markdownlint-disable-file MD013 MD036 MD041 -->
+**Installing dependencies**
+
+The python snippets require the [dasbus](https://dasbus.readthedocs.io/en/latest/) package. Install it via:
+
+```bash
+pip install dasbus
```

Comment:
Can't we use RPM package instead?

Reply:
responded https://github.com/containers/bluechi/pull/563#discussion_r1325752617